xtuner/v1/engine/train_engine.py

```diff
@@ -77,7 +77,7 @@ def __init__(self, model_path, cache_dir=None, from_hub="huggingface"):
             self.use_safetensors = False
         elif "model.safetensors" in os.listdir(self.model_path):
             with safe_open(os.path.join(self.model_path, "model.safetensors"), framework="pt") as f:
-                self.weight_map = {k: "model.safetensors" for k in f.keys()}
+                self.weight_map = dict.fromkeys(f.keys(), "model.safetensors")
```

Comment:
This change looks good to me and is fine to include in this PR. However, I'd suggest avoiding modifications to unrelated code in future PRs to keep changes focused and easier to review.